doc/tutorial/03_index/index.md

```diff
@@ -237,43 +237,86 @@ does not need any special parameters.
 
 ## IBF vs HIBF {#hibf}
 
-/todo This is a placeholder section and needs more information including the chopper layout.
-
 Raptor works with the Interleaved Bloom Filter by default. A new feature is the Hierarchical Interleaved Bloom Filter
-(HIBF) (raptor::hierarchical_interleaved_bloom_filter), which can be used with `--hibf` can be used. This uses a more
+(HIBF) (raptor::hierarchical_interleaved_bloom_filter), which can be used with `--hibf`. This uses a more
 space-saving method of storing the bins. It distinguishes between the user bins, which reflect the individual samples as
 before, and the so-called technical bins, which throw some bins together. This is especially useful when there are
 samples of very different sizes.
+
+To use the HIBF, a layout must be created before creating an index. We have written an extra tutorial for this
+\ref tutorial_layout.
+
+### HIBF indexing with the use of the layout
+
+The layout replaces the `all_bin_path.txt` and is given instead with the HIBF parameter: `--hibf binning.out`.
+
 Since the HIBF calculates the size of the index itself, it is no longer possible to specify a size here. But we can
-offer the option to name the desired false positive rate with `--fpr`.
+offer the option to name the desired false positive rate with `--fpr`. Thus, for example, a call looks like this:
 
 ```bash
-raptor build --kmer 19 --hash 3 --hibf --fpr 0.1 --output raptor.index all_bin_paths.txt
+raptor build --hibf binning.layout \
+             --kmer 16  \
+             --window 20  \
+             --hash 3 \
+             --fpr 0.25  \
+             --threads 2 \
+             --output hibf.index
+```
+
+\note wichtig!!! --false-positive-rate "$fpr"  muss die selbe sein wie die von raptor Genauso die k-mer size. Hash functions auch oder?
+\warning test
```

Comment:
Yes. Later, raptor build should read the parameters from the layout file. And store them in the index such that `search` doesn't need the fpr, etc.

Passing wrong fpr/hash functions would lead to different false positive rates in the results, though in principle the algorithm still works.

Reply:
So we will just leave it uncommented?

Reply:
Probably yes, for now.